plugins/grpc/swift/v1.19.1/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1.4
```

Comment:
I wonder if we should change this to 

```suggestion
# syntax=docker/dockerfile:1
```

Or have a post-process to update based on the latest stable frontend Dockerfile.

@pkwarren any prefernce?